src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task07.java

```diff
@@ -36,8 +36,10 @@ public static void main(String[] args) {
      * @return <code>false</code>/<code>true</code>
      */
     static boolean isLeapYear(int year) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return false;
+        if ((year % 400 == 0) || (year % 4 == 0) && (year % 100 != 0)){
```

Comment:
"Violation: There isn't separating white spaces for condition
Solution: Add white spaces"